lib/ibandit/swedish_details_converter.rb

```diff
@@ -32,6 +19,62 @@ def self.convert(account_number)
       }
     end
 
+    private
+
+    def cleaned_account_number
+      @cleaned_account_number ||= clean_number(@account_number)
+    end
+
+    def cleaned_branch_code
+      @cleaned_branch_code ||= clean_number(@branch_code)
+    end
+
+    def clean_number(number)
+      return nil if number.nil?
+      number.gsub(/[-.\s]/, '').gsub(/\A0+/, '')
+    end
+
+    def bank_info
+      @bank_info ||= self.class.bank_info_for(bank_info_key)
```

Comment:
Sorry, should comment on PR diff instead of commit because GH is shit at commit comments. I think I'd prefer to have `bank_info_for` inlined here.